LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+TODO: Actual Authors Here
```

Comment:
This does not look like `...and include correct authors`

Reply:
Agree! I'm not sure that this project is licensed under MIT license, so let's wait for actual authors answers!